src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import core.basesyntax.service.DataMaker;
+import core.basesyntax.service.DataReader;
+import core.basesyntax.service.DataWriter;
+import core.basesyntax.service.LineParser;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.DataMakerImpl;
+import core.basesyntax.service.impl.DataReaderImpl;
+import core.basesyntax.service.impl.DataWriterImpl;
+import core.basesyntax.service.impl.LineParserImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.strategy.FruitTransactionStrategy;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        DataReader<String> dataReader = new DataReaderImpl();
+        String fromFile = "src/main/resources/inputData.csv";
+        List<String> inputData = dataReader.dataFromFile(fromFile);
+        LineParser lineParser = new LineParserImpl();
+        List<FruitTransaction> fruitTransactions = lineParser.createListOfTransactions(inputData);
+        FruitTransactionStrategy transactionStrategy = new FruitTransactionStrategy();
+        transactionStrategy.executeTransaction(fruitTransactions);
+        ReportService reportService = new ReportServiceImpl();
+        String report = reportService.makeReport();
+        DataMaker dataMaker = new DataMakerImpl();
+        String data = dataMaker.getDataToWrite(report);
```

Comment:
What is the difference between **report** and **data**? Why ReportService does not provide us ready for use report?